ucsb-cs156-s22/s22-4pm-courses
# 4pm-2 - kt - psid sections endpoint

In this PR, we add the endpoint to get section information given a personal schedule id.
Note: This PR is an update of PR #53  , since I had to use a new branch to get the updates done in main.
Closes Issue #22 

Endpoint: /api/personalSections/all
<img width="1256" alt="image (4)" src="https://user-images.githubusercontent.com/34698996/172028252-d2245896-75ec-4e90-a608-96f7599bbc65.png">

Test plan:
1. Create/ensure that a personal schedule exists. Note the id.
2. Create/ensure there are courses associated with the personal schedule from earlier (the psid field of courses references the id field of personal schedules).
3. The personal sections endpoint, when given the personal schedule id, should output the courses associated with the schedule, formatted in section layout defined by the Course documents.
